action.yml

```
@@ -6,6 +6,8 @@ inputs:
     description: 'Installation URL that will contain a script to install Nix.'
   install_options:
     description: 'Additional installer flags passed to the installer script.'
+  single_user:
+    description: 'Always install Nix in singer-user mode'
```

Was just searching about some bug, ended up reading this.
```suggestion
    description: 'Always install Nix in single-user mode'
```